content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/vmtransform.rb

```diff
@@ -32,8 +32,8 @@ def main
               @handle.log(:info, "Transformation - Started On: #{start_timestamp}")
 
               destination_ems = @handle.vmdb(:ext_management_system).find_by(:id => task.get_option(:destination_ems_id))
-              max_runners = destination_ems.custom_get('Max Transformation Runners').to_i || factory_config['max_transformation_runners_by_ems'] || 10
-              if Transformation::TransformationHosts::Common::Utils.get_runners_count_by_ems(destination_ems, factory_config) >= max_runners
+              max_runners = destination_ems.custom_get('Max Transformation Runners') || factory_config['max_transformation_runners_by_ems'] || 10
```

Comment:
Same concepts apply here as well.

Reply:
I'm using the embedded method to not duplicate code. A separate method is even a better idea because the default value lives in a single method.